### ISSUE TITLE
Visit Page Search Flow

### ISSUE THREAD

**Issue by skyheat** [codersforcauses/poops#272]
**Is your feature request related to a problem? Please describe.**
**Blocked by #271 **
Currently there is no search flow for the visit page. 

**Describe the solution you'd like**
Implement a flow where the user is taken to a new page with inputs for client name and a date range picker.

**Describe alternatives you've considered**

**Additional context**

**Comment by scratchclaggy** [comment]
![image](https://user-images.githubusercontent.com/34858205/219845611-1a65a3b3-c782-4ebc-a56d-15299ee6ec0d.png)

`src/pages/visit/index.tsx`

We need:
- startDate
- endDate
- clientName

**Comment by NicholasChoong** [comment]
we have fixed this in another issue
